src/pre_process/m_initial_condition.fpp

```diff
@@ -284,6 +288,9 @@ contains
                 elseif (patch_ib(i)%geometry == 4) then
                     call s_airfoil(i, ib_markers%sf, q_prim_vf, .true.)
```

Comment:
weird subroutine name. s_airfoil_XXX what does this subroutine do?

Reply:
It's an analytical airfoil patch. I believe @anandrdbz created this patch. Can I make a separate PR specifically for refactoring something like this later?

Reply:
please just fix it now, it should only change a couple of lines?